ResearchKit/Common/ORKTaskViewController.m

```diff
@@ -946,19 +949,18 @@ - (void)showViewController:(ORKStepViewController *)viewController goForward:(BO
     
     ORKWeakTypeOf(self) weakSelf = self;
     [self.pageViewController setViewControllers:@[viewController] direction:direction animated:animated completion:^(BOOL finished) {
-        ORKStrongTypeOf(weakSelf) strongSelf = weakSelf;
```

Comment:
That's a good thought. I'll try that solution and see if the crash still exists.  It would be a much smaller change.